internal/converter/converter.go

```diff
@@ -282,6 +282,11 @@ func (c *Converter) convertFile(file *descriptor.FileDescriptorProto, fileExtens
 			}
 			enumJSONSchema.Version = c.schemaVersion
 
+			if len(file.GetMessageType()) == 0 {
+				pkgName := file.GetPackage()
+				enumJSONSchema.FullRef = fmt.Sprintf("%s%s.%s", c.refPrefix, pkgName, enum.GetName())
```

Comment:
optional: We could not include the ref prefix if we're going to strip it out later anyways. If we go this route, we could also rename this from FullRef to something like `ProtobufPath`.

Reply:
we could but it would probably make sense to modify all refs (field types are also refs) and i'm not sure we want to do that ☹️